domain/schema/model/sql/0004-read-only-model.sql

```diff
@@ -16,7 +16,8 @@ CREATE TABLE model (
     cloud_region TEXT,
     credential_owner TEXT,
     credential_name TEXT,
-    is_controller_model BOOLEAN DEFAULT FALSE
+    is_controller_model BOOLEAN DEFAULT FALSE,
+    environ_version INT DEFAULT 0
```

Comment:
This is the read only model, if we ever implement environment upgrader, this can't be here, as it's per the name - read only.

Reply:
All fine we have discussed this. We don't need any DB changes to support this working :)

Reply:
Thanks Simon and Tom. Yup tom and I have discussed this and will proceed to close this PR